src/acl/agent.test.ts

```diff
@@ -911,6 +913,101 @@ describe("setAgentResourceAccess", () => {
     );
   });
 
+  it("adds the appropriate Quads for the given Access Modes if the rule is both a resource and default rule", async () => {
+    const sourceDataset = Object.assign(dataset(), {
+      internal_accessTo: "https://arbitrary.pod/resource",
+      internal_resourceInfo: {
+        fetchedFrom: "https://arbitrary.pod/resource/?ext=acl#owner",
+        isLitDataset: true,
+      },
+    });
+    sourceDataset.add(
+      DataFactory.triple(
+        namedNode("https://arbitrary.pod/resource/?ext=acl#owner"),
+        namedNode("http://www.w3.org/1999/02/22-rdf-syntax-ns#type"),
+        namedNode("http://www.w3.org/ns/auth/acl#Authorization")
+      )
+    );
+    sourceDataset.add(
+      DataFactory.triple(
+        namedNode("https://arbitrary.pod/resource/?ext=acl#owner"),
+        namedNode("http://www.w3.org/2000/01/rdf-schema#label"),
```

Comment:
Do we really need an `rdfs:label` triple in this test data...?

Reply:
It's not necessary, but it was part of the data that prompted the bug, so I wanted the test to be as accurate as possible. It has no effect in this case, so it could be removed, but I was wondering if having real-world data in our tests wasn't an additional safety rail against regression.

Reply:
I'd remove it. 
I don't understand what you mean about 'accurate as possible', or 'real-world data' here - i.e. if the original bug report also contained an 'rdfs:comment' triple and 15 'foaf:knows' triples, would you also include them in this test?! I understand you'd include them initially when trying to find out where the actual bug is, but once you understand and fix the bug, then the test data should be cleaned up to only contain whatever data is necessary to test the specific condition.

Reply:
Fixed in a9d6a522e9c5964b22392397c1e2492b2217fa28.